frontend/webadmin/modules/uicompat/src/main/java/org/ovirt/engine/ui/uicompat/UIConstants.java

```diff
@@ -2246,4 +2246,6 @@
     String confirmSuspendingVm();
 
     String homePageCustom();
+
+    String statelessVmFieldDisabledReason();
```

Comment:
the name is too general, no? maybe statelessDisabledForLatestTemplateVersionReason?

Reply:
I think it is OK, we do not necessarily use separate message for each condition and we use the general label quite often:
https://github.com/oVirt/ovirt-engine/blob/5de3d52335c9b2e65d52915eda9696cca4c14340/frontend/webadmin/modules/uicompat/src/main/resources/org/ovirt/engine/ui/uicompat/UIConstants.properties#L1133
https://github.com/oVirt/ovirt-engine/blob/5de3d52335c9b2e65d52915eda9696cca4c14340/frontend/webadmin/modules/uicompat/src/main/resources/org/ovirt/engine/ui/uicompat/UIConstants.properties#L1136

Reply:
ok